pkg/providers/azure.go

```diff
@@ -2,9 +2,14 @@ package providers
 
 import (
 	"context"
+	"encoding/base64"
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
```

Comment:
this seems like an outdated version of this package
https://pkg.go.dev/github.com/azure/azure-sdk-for-go@v63.2.0+incompatible/profiles/latest/graphrbac/graphrbac

Reply:
looks like we don't actually want to be using this package at all.

https://github.com/Azure/azure-sdk-for-go/issues/20660